### PR TITLE
Chico 1071 - Dynamic Envelope Stack Rendering

### PR DESCRIPTION
**Pull request by BFisch14**

### CODE REVIEW

File: react/envelope_stack.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react'
-import {ReactComponent as ToDoFull} from './assets/stack_to-do_full.svg';
+import {ReactComponent as ToDoFull} from './assets/stack_to-do_half.svg';
```
Comment (ramcguire) [inline react/envelope_stack.js:2]: Do you want the ToDoFull component to take from stack_to-do_half.svg?

File: react/envelope_stack.js [hunk 2]
```diff
@@ -81,23 +81,43 @@ class EnvelopeStack extends Component {
       count = 0;
     }
     return (
-      <div style={{width: "15%"}}>
-        {this.props.stackType === 0 ? "Ready" : "Finished"}
-        <br></br>
+      <div style={{width: "15%", position: "relative"}}>
+        {/* {this.props.stackType === 0 ? "Ready" : "Finished"}
```
Comment (ramcguire) [inline react/envelope_stack.js:85]: Should this be commented out?

**Review by ramcguire — COMMENTED** [review]
Made a couple comments on parts of the code, make sure that is the behavior you want.

File: react/envelope_stack.js (hunk 1 above)
Comment (BFisch14) [reply react/envelope_stack.js]: Fixed

File: react/envelope_stack.js (hunk 2 above)
Comment (BFisch14) [reply react/envelope_stack.js]: Fixed

**Review by ramcguire — APPROVED** [review]
Looks good!

**Pull request merged**